requirements-dev.txt

```diff
@@ -1,9 +1,11 @@
+attrs>=19.2.0
 coveralls
 pytest>=4.6
 pytest-cov
 pytest-trio>=0.5.0
 sphinx
 sphinxcontrib-trio
 sphinx_rtd_theme
+trio>=0.15.0
```

Comment:
oh right-- because of Trio API upgrade in the tests, we can no longer do a Python 3.5 build 😿

Reply:
If this causes trouble then the commit can be reverted and re-applied once Python 3.5 is dropped.

Reply:
I think I'll just add fallback to hazmat for now